quickstarts/contentsquare/config.yml

```diff
@@ -0,0 +1,69 @@
+# Name of the quickstart (required)
+name: contentsquare
+
+# Displayed in the UI (required)
+title: Contentsquare
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Contentsquare is a powerful, yet easy-to-use, digital experience analytics 
+  platform that collects and quantifies user behavior across all your digital 
+  properties showing you what users like, what they don’t like, where they 
+  struggle, and why they leave. 
+  Contentsquare surfaces opportunities to improve your customer experience
+  automatically and easily so you can increase your conversion rates, improve
+  customer loyalty, reduce costs and drive revenue.
+
+# Long-form description of the quickstart (required)
+description: |
+  Get more value from your investments in Contentsquare and New Relic by seeing
+  how performance issues truly affect your customers’ digital experience. 
+
+  Through this integration, access Contentsquare Session Replay links directly
+  within New Relic. With this, Contentsquare shows you exactly what a user was 
+  doing before, during, and after an issue occurs, so that you can remediate 
+  issues quickly and with the complete picture. Dramatically reduce the 
+  time-to-repair of system errors by linking to your impacted users' exact 
+  session replays.
+
+  - <b>Reduce time-to-recovery</b>
```

Comment:
@jpvajda quick question for confirmation: is html formatting accepted?

Reply:
@vincentdestrait It's markdown, so HTML elements can work, but your best approach would be to use **markdown bolding** as that is traditional markdown.

https://www.markdownguide.org/cheat-sheet/
https://stackoverflow.com/questions/40335929/including-html-in-markdown

Reply:
@jpvajda perfect thank you, I've just updated the syntax then